services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -305,11 +315,16 @@ async def _process_task_output(self):
             LogType.LOG, "[sidecar]Uploading outputs...",
         )
         PORTS = await self._get_node_ports()
+        STEM_OUTPUTS = (
+            "output"
```

Comment:
this is ok for now but we should think in an abstraction that would allow having full control of compatibility. Probably this should be centralized such that we could automaticaly generat e.g. a compatilbility matrix between services metadata and sidecar version. Something similar to https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix

Reply:
agree